src/ServiceControl.Config/Extensions/Validations.cs

```diff
@@ -55,9 +62,10 @@ public static List<string> UsedQueueNames(this ReadOnlyCollection<ServiceControl
                 .ToList();
         }
 
-        public static List<string> UsedQueueNames(this ReadOnlyCollection<ServiceControlAuditInstance> ServiceControlInstances, TransportInfo transportInfo = null, string instanceName = null, string connectionString = null)
+        public static List<string> UsedAuditQueueNames(TransportInfo transportInfo = null, string instanceName = null, string connectionString = null)
         {
-            var instancesByTransport = ServiceControlInstances.Where(p => p.TransportPackage.Equals(transportInfo) &&
+            var serviceControlInstances = InstanceFinder.ServiceControlInstances();
```

Comment:
Does this need to take into account ServiceControl Audit instances?

Reply:
Good catch. That was actually incorrect. Updated.